package.json

```diff
@@ -27,7 +26,7 @@
 		"nodemon": "^1.18.5"
 	},
 	"scripts": {
-		"install": "lerna bootstrap",
-		"test": "lerna run test"
+		"install": "cd json-lint; npm install",
```

Comment:
I think this is going to break the linter as we will not install dependencies inside this package.json. I would keep things simple and just have one package json (this one) with all the dependencies (json lint has 2 dependencies).
Also we should remove eslint and eslint-plugin-node from devDependencies